.github/workflows/scorecards.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       # Upload the results as artifacts (optional). Commenting out will disable uploads of run results in SARIF
       # format to the repository Actions tab.
       - name: "Upload artifact"
-        uses: actions/upload-artifact@6673cd052c4cd6fcf4b4e6e60ea986c889389535 # tag=v3.0.0
+        uses: actions/upload-artifact@v3 # tag=v3.0.0
```

Comment:
I'm not sure why scorecards needed a specific hash commit,
but this looks okay to me.

cc @felixhandte